disaster-recovery/src/main/scala/uk/gov/nationalarchives/disasterrecovery/Processor.scala

```diff
@@ -207,6 +207,7 @@ class Processor(
           removeFileExtension(bitStreamInfo.name)
         )
       } ++ metadata
+    case None => IO.pure(Nil)
```

Comment:
Shouldn't this be an error or should we be alerted to this somehow?

Reply:
`None` is when we get a message for a structural object. I did it this way to avoid having another case class for a StructuralObject that wasn't going to be used. In this instance, we don't want to do anything but we still want to delete the message.